ui/src/widgets/ui-switch/UISwitch.vue

```diff
@@ -113,7 +136,7 @@ export default {
         },
         toggle () {
             if (this.state.enabled) {
-                if (this.props.decouple) {
+                if (this.getProperty('decouple')) {
```

Comment:
Better to move this to a computed property and refer it here as a readability improvement. But we can do that later

```
computed: {
    decouple () {
        return this.getProperty('decouple')
    }
}
```

Reply:
> are we setting Passthru as a dynamic property https://github.com/FlowFuse/node-red-dashboard/issues/1181. It's not implemented other widgets at the moment

It wasn't a priority in others, switch is used in different ways where having this would be useful. Not goign to push it away if it's already been implemented!

Reply:
Thanks for the feedback Joe. I will check this separately